src/plugins/intel_cpu/src/nodes/softmax.cpp

```diff
@@ -182,7 +181,7 @@ void SoftMax::prepareParams() {
             key.attr,
             true);
 
-        primitive_desc_iterator itpd = *desc;
+        primitive_desc_iterator itpd = prim_desc;
 
         while (itpd) {
             impl_desc_type impl_type = parse_impl_name(itpd.impl_info_str());
```

Comment:
@zhangYiIntel May I ask you to take a look at this?
Will it be correct to re-design this logic the same way it is done for matmul, i.e. just pick first available implementation in case key.implType == ref_any?

Reply:
Yeah, that is why I have not touched it. Lets leave it for now.